lib/command.c

```diff
@@ -2507,7 +2527,12 @@ void cmd_init(int terminal)
 	cmdvec = vector_init(VECTOR_MIN_SIZE);
 
 	/* Default host value settings. */
-	host.name = NULL;
+	host.name = XSTRDUP(MTYPE_HOST, names.nodename);
+#ifdef HAVE_STRUCT_UTSNAME_DOMAINNAME
+	host.domainname = XSTRDUP(MTYPE_HOST, names.domainname);
```

Comment:
The check for `(none)` should be here, so we get NULL instead of "(none)"

Reply:
sure. Made the changes.